pkgs/development/libraries/crypto++/default.nix

```diff
@@ -1,49 +1,41 @@
-{ fetchFromGitHub, stdenv }:
+{ stdenv, fetchFromGitHub }:
```

Comment:
to fix your eval
```suggestion
{ stdenv, fetchFromGitHub, libtool }:
```

Reply:
Yes indeed, pushed too quickly before leaving home :p
I'll correct it later today.